### ISSUE TITLE
Finetuning a lora

### ISSUE THREAD

**Issue by christopher5106** [kohya-ss/musubi-tuner#909]
Hi,

I'm trying to finetune on high quality dataset, a lora that I have previously trained on a large low quality dataset. And I'm surprised to see that at first step I don't recover the results of initial lora.

First I tried with `--resume` option, then I tried to load myself into `accelerator._models[1]`, last I tried the `--network_weights` option. All of them don't start with initial lora.

Please help and advise if you have any ideas.

**Comment by GreenLandisaLie** [comment]
Use --base_weights but then merge the output with all networks used in --base_weights and using the same rations as set by --base_weights_multiplier (1.0 by default).

To test this out try training for at least 100 steps and use the new lora alongside the ones in --base_weights in inference.

EDIT:
Just to be clear, this is only meant to be done when training on top of different (or multiple) loras. When resuming from a training you should use --resume if nothing changed or --network_weights if you did change something major. --network_weights is more lenient to changes than --resume - but even so it still does not allow you to change the type and rank of the lora and changing the optimizer is also either not allowed or it would probably break your lora.

**Comment by christopher5106** [comment]
Thank you for the solution with `--base_weights`, indeed it looks great.

But just by curiosity, I was a bit surprised not to be able to finetune a lora... 

- I tried to resume from a training state, but a different number of epochs and the sample images were broken after first epoch

- I tried to load (with torch) directly the model.safetensors from the state (avoiding to resume the optimizer state) but sampling was still broken at first epoch.

- I tried `--network_weights`, but results were still broken after first epoch.

Does that mean that lora training only works from zero initialization ? or is it a wrong conclusion ?

**Comment by kohya-ss** [comment]
You should be able to train LoRA with `--network_weights`. Please specify the `--network_weights` and `--sample_at_first` options to check whether LoRA is loaded in the pre-training sample. If the image is not corrupted, LoRA in `--network_weights` is loaded correctly.

**Comment by christopher5106** [comment]
So, i tried with `--sample_at_first --network_weights`, the first sample outcome corresponds to my previously trained lora, but for the sample after the first epoch, the image is completely exploded. We can still recognized the character it has been trained on from details such as the beard, but it's a broken image I never saw during any lora training. Does that mean loras training cannot start from zero intialization ?

Looks like the only way is to use `--base_weights`

**Comment by kohya-ss** [comment]
If the first sample is correct, LoRA should be reading it correctly. Could you please try lowering the learning rate to something like 1/10 or 1/100 of the current value?

**Comment by christopher5106** [comment]
Yes it helps. I think it's me, I'm  no more used to low quality training in 10 epochs on 10 images where it destroys everything at first epoch :) 
I got so good loras with bigger datasets with higher number of epochs
